Runs: Rewrite Runs section (#2011)
wandb/docs: models/runs/tags.mdx

## Remove tags from one or more runs

> Tags can also be removed from runs with the W&B App.

Reviewer: ```suggestion
Follow these steps to remove tags from a run in the W&B App.
```
